### PR TITLE
Add PHP_DATETIMEZONE as ENV Variable

### PR DESCRIPTION
**Pull request by holzi1005**
Using a new ENV to set the date_timezone in custom.ini for php-fpm.

The previous Pull Request has some issues with the formating, so I added a new one with only the three requested changed files.

Regards,
holzi1005

## Summary by CodeRabbit

- **New Features**
	- Introduced a new environment variable `date_timezone` with a default value of `UTC` for enhanced timezone configuration in PHP.
	- Expanded instructions in the README for adding Composer to the Docker image, including a new example for efficient dependency management.

- **Bug Fixes**
	- Updated the PHP configuration to use the dynamic variable `${date_timezone:-UTC}` for timezone settings instead of a hardcoded value.

- **Documentation**
	- Enhanced the README with clearer instructions and formatting improvements for better readability.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -95,4 +95,5 @@ ENV nginx_root_directory=/var/www/html \
     memory_limit=128M \
     post_max_size=8M \
     upload_max_filesize=2M \
-    zlib_output_compression=On
+    zlib_output_compression=On \
+    PHP_DATETIMEZONE=UTC
```
[the remark this answers is not in the document]
Comment (holzi1005) [reply Dockerfile:99]: Fixed in README

File: README.md [hunk 2]
```diff
@@ -167,7 +167,8 @@ You can define the next environment variables to change values from NGINX and PH
...
-| PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |
+| PHP8   | zlib_output_compression | On            | Whether to transparently compress pages. If this option is set to "On" in php.ini or the Apache configuration, pages are compressed if the browser sends an "Accept-Encoding: gzip" or "deflate" header.                                               |                                                                         |
+| PHP8   | PHP_DATETIMEZONE        | UTC           | Sets the PHP timezone configuration (date.timezone) in custom.ini. Accepts standard PHP timezone identifiers (e.g., 'America/New_York', 'Europe/London'). See [PHP timezones](https://www.php.net/manual/en/timezones.php) for valid values. |
```
[the remark this answers is not in the document]
Comment (holzi1005) [reply README.md]: fixed

**Review by erseco — CHANGES_REQUESTED** [review]
Hi @holzi1005,

Thank you for submitting the changes and the new Pull Request. It’s great to see the updates, especially the introduction of the new `PHP_DATETIMEZONE` environment variable and the improved flexibility for timezone configuration in `custom.ini`.

I would like to emphasize the importance of maintaining consistent code style throughout the project. Specifically, in the `alpine-php-webserver`, all PHP variables should be lowercase as per the established code standards. Please ensure this is reflected for any future additions or updates.

Thank you for addressing the formatting issues from the previous pull request and keeping the changes focused on the requested files.

Best regards,  
Ernesto

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (holzi1005) [reply Dockerfile]: I just updated it.
The best practice is to make all env capital but I will change them the lower case.

**Review by erseco — APPROVED** [review]

**Comment by erseco** [comment]
Thanks for your contribution!

**Comment by erseco** [comment]
Thanks for your contribution!

**Comment by holzi1005** [comment]
@erseco When do you create a new release version so I can add the image to my docker files?

**Comment by erseco** [comment]
> @erseco When do you create a new release version so I can add the image to my docker files?

I generated the release a few minutes ago! :)

**Pull request merged**